sdk/database/dbplugin/v5/grpc_client.go

```diff
@@ -121,6 +121,7 @@ func newUserReqToProto(req NewUserRequest) (*proto.NewUserRequest, error) {
 		CredentialType: int32(req.CredentialType),
 		Password:       req.Password,
 		PublicKey:      req.PublicKey,
+		Subject:        req.Subject,
```

Comment:
Wanted to note that we should manually test running the mongodb atlas plugin as an external plugin before merging this. I remember hitting some non-obvious errors when introducing new fields for the snowflake keypair credential type. Happy to help with this!